homeassistant/components/zwave_js/config_flow.py

```diff
@@ -707,7 +717,7 @@ def _async_update_entry(self, data: dict[str, Any]) -> None:
 
     async def async_step_init(
         self, user_input: dict[str, Any] | None = None
-    ) -> FlowResult:
+    ) -> ConfigFlowResult:
```

Comment:
Do we need a different result for options flows?

Reply:
Yes, we should probably have that, but there are a handful of integrations which use the same base class for config and option flows and I couldn't work out a way to make the type checker accept it.

Should we give it another try before merging this PR or do you think it's OK to either let config and option flow share the return type or try to work it out in a follow-up?

Reply:
We can do it in a follow up.